src/solvers/refinement/README.md

```diff
@@ -6,6 +6,22 @@
 
 \section string_solver_interface String solver interface
 
+After the input program of CProver has been converted to a goto-program and then
+to a set of equations, these equations are fed to a solver.
```

Comment:
This is what bounded model checking does, but other verification techniques might use a different approach?